espnet2/fileio/rttm.py

```diff
@@ -0,0 +1,119 @@
+# Licensed under the MIT license.
+
+import collections.abc
+import humanfriendly
+from pathlib import Path
+from typing import Dict
+from typing import List
+from typing import Mapping
+from typing import Set
+from typing import Tuple
+from typing import Union
+
+import numpy as np
+import re
+from pathlib import Path
+from typeguard import check_argument_types
+
+from espnet2.fileio.read_text import read_2column_text
+
+
+def load_rttm_text(
+    path: Union[Path, str]
+) -> (Dict[str, List[Tuple[str, float, float]]], List[str]):
+    """Read a RTTM file
+
+    Note: only support speaker information now
+    """
+
+    assert check_argument_types()
+    spk_index = 0
+    data = {}
+    spk_dict = {}
+    with Path(path).open("r", encoding="utf-8") as f:
+        for linenum, line in enumerate(f, 1):
+            sps = re.split(" +", line.rstrip())
+
+            # RTTM format must have exactly 9 fields
+            assert len(sps) == 9 and path
```

Comment:
Could you always add a message after assertion check to show the status for debugging?

```python
assert len(sps) == 9 and path, len(sps)
```

What `and path` is?

Reply:
Just mean to show the files in the wrong format. I will attach the message letter.